django/contrib/admin/actions.py

```diff
@@ -33,8 +33,11 @@ def delete_selected(modeladmin, request, queryset):
 
     # Populate deletable_objects, a data structure of all related objects that
     # will also be deleted.
-    deletable_objects, model_count, perms_needed, protected = get_deleted_objects(
-        queryset, opts, request.user, modeladmin.admin_site, using)
+    deletable_objects, model_count, perms_needed, protected = (
```

Comment:
@MarkusH I don't have a strong opinion on this, but I wouldn't classify this change as a  backwards-incompatible, I guess.

@rebkwok Maybe we can keep previous indentation i.e.:
``` python
deletable_objects, model_count, perms_needed, protected = modeladmin.get_deleted_objects(
    queryset, opts, request, using)
```